celavi/pylca_celavi/pylca_opt_foreground.py

```diff
@@ -26,12 +26,13 @@
 
     
 #Reading in static and dynamics lca databases
-print('>>>', os.getcwd())
-df_dynamic = pd.read_csv('dynamic_secondary_lci_foreground.csv')
+
 
 #We are integrating static lca with dynamics lca over here. 
-def preprocessing(year,df_static):
+def preprocessing(year,df_static,dynamic_lci_filename):
     
+    #Reading in dynamics LCA databases
+    df_dynamic = pd.read_csv(dynamic_lci_filename)
```

Comment:
This particular function removes any dummy flows from the original USLCI if they exist. USLCI has a lot of dummy flows. Now in Opt foreground, we dont use USLCI. We use our own made inventory. However, if there is a mistake in the inventory and someone adds a product flow without a process ( mistake in building the file this will solve the issue) Do we need a warning for that?

Reply:
Yes, if the warning indicates that the foreground inventory was not specified correctly then a warning would be good to have. Unlikely we'll need it for the wind study but it'll be useful in the future.